Packs/MicrosoftGraphMail/Integrations/MicrosoftGraphListener/README.md

```diff
@@ -678,4 +680,12 @@ Delete a specific email rule by ID for a user's mailbox using Microsoft Graph AP
 
 #### Context Output
 
-There is no context output for this command.
\ No newline at end of file
+There is no context output for this command.
+
+## Troubleshooting
+
+In case of **hash verification** error:
+- Use the Oproxy flow to generate a new pair of credentials. This is crucial as it ensures that any issues related to authentication can be mitigated with fresh credentials.
+- Execute the command *!msgraph-mail-auth-reset*. This command resets the authentication mechanism, allowing for the new credentials to be accepted.
+- Insert the newly created credentials into the original instance where the error occurred. Make sure the credentials are entered correctly to avoid further errors.
+- After updating the credentials, test the integration.
```

Comment:
```suggestion
1. Use the Oproxy flow to generate a new pair of credentials. This is crucial as it ensures that any issues related to authentication can be mitigated with fresh credentials.
2. Execute the command ***!msgraph-mail-auth-reset***. This command resets the authentication mechanism, allowing for the new credentials to be accepted.
3. Insert the newly created credentials into the original instance where the error occurred. Make sure the credentials are entered correctly to avoid further errors.
4. After updating the credentials, test the integration.
```